cmd/api-headers.go

```diff
@@ -83,6 +83,20 @@ func setPartsCountHeaders(w http.ResponseWriter, objInfo ObjectInfo) {
 	}
 }
 
+// Modif parts content-lengh
+func setPartsContenLengtHeaders(w http.ResponseWriter, objInfo ObjectInfo, partNumber int) {
+	if strings.Contains(objInfo.ETag, "-") && len(objInfo.Parts) > 0 {
```

Comment:
This duplicates code in `setPartsCountHeaders`. Remove one or the other and only have one function call.